### ISSUE TITLE
[JENKINS-25836] Cannot add steps that are available through drop down menus

### ISSUE THREAD

**Issue by jenkins-infra-bot** [timja-org/jenkins-gh-issues-poc-11-15#12299]
<p>We upgraded to 1.580.1 and we are seeing a strange bug that happens after jenkins has run for few mins. We cannot add any steps that are available as part of drop down menus. e.g I can't add a new post build action to an existing job. This doesn't happen right after we restart jenkins, but after jenkins has run for few mins we are seeing this issue. This issue is not isolated to any plugin rather we are seeing this everywhere.</p>

<p>when debugged using dev console, we are seeing this 404 error.</p>

<p> POST <a href="http://ewe.builds.sb.karmalab.net/$stapler/bound/ee41fccd-031c-4740-aa2d-d74412a9523e/render" class="external-link" target="_blank" rel="nofollow noopener">http://ewe.builds.sb.karmalab.net/$stapler/bound/ee41fccd-031c-4740-aa2d-d74412a9523e/render</a> 404 (Not Found)</p>

<p>prototype.js:1585 </p>

<p>Ajax.Request.Class.create.request     prototype.js:1550 Ajax.Request.Class.create.initialize       prototype.js:452 <br/>
(anonymous function)prototype.js:101 <br/>
klass       bind.js:51 <br/>
proxy.(anonymous function)       hudson-behavior.js:540 <br/>
renderOnDemand hetero-list.js:53 <br/>
(anonymous function) event-min.js:7 <br/>
YAHOO.util.CustomEvent.notify    event-min.js:7 <br/>
YAHOO.util.CustomEvent.fire menu-min.js:7    <br/>
e   event-min.js:7 <br/>
n</p>

<p>I have also attached the screenshot of the console.</p>

<p>Any thoughts on what could be causing this?</p>

---
<details><summary><i>Originally reported by <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=kuldeeps">kuldeeps</a>, imported from: <a href="https://issues.jenkins.io/browse/JENKINS-25836" target="_blank">Cannot add steps that are available through drop down menus</a></i></summary>
<i><ul>
<li><b>status</b>: Resolved
<li><b>priority</b>: Blocker
<li><b>component(s)</b>: core
<li><b>label(s)</b>: jenkins
<li><b>resolution</b>: Cannot Reproduce
<li><b>resolved</b>: 2015-03-17T20:13:29+00:00
<li><b>votes</b>: 1
<li><b>watchers</b>: 4
<li><b>imported</b>: 2025-11-16
</ul></i>
<details><summary>Raw content of original issue</summary>

<pre>
<p>We upgraded to 1.580.1 and we are seeing a strange bug that happens after jenkins has run for few mins. We cannot add any steps that are available as part of drop down menus. e.g I can't add a new post build action to an existing job. This doesn't happen right after we restart jenkins, but after jenkins has run for few mins we are seeing this issue. This issue is not isolated to any plugin rather we are seeing this everywhere.</p>

<p>when debugged using dev console, we are seeing this 404 error.</p>

<p> POST <a href="http://ewe.builds.sb.karmalab.net/$stapler/bound/ee41fccd-031c-4740-aa2d-d74412a9523e/render" class="external-link" target="_blank" rel="nofollow noopener">http://ewe.builds.sb.karmalab.net/$stapler/bound/ee41fccd-031c-4740-aa2d-d74412a9523e/render</a> 404 (Not Found)</p>

<p>prototype.js:1585 </p>

<p>Ajax.Request.Class.create.request     prototype.js:1550 Ajax.Request.Class.create.initialize       prototype.js:452 
(anonymous function)        prototype.js:101 
klass       bind.js:51 
proxy.(anonymous function)       hudson-behavior.js:540 
renderOnDemand         hetero-list.js:53 
(anonymous function)         event-min.js:7 
YAHOO.util.CustomEvent.notify    event-min.js:7 
YAHOO.util.CustomEvent.fire         menu-min.js:7    
e           event-min.js:7 
n</p>

<p>I have also attached the screenshot of the console.</p>

<p>Any thoughts on what could be causing this?</p>

</pre>
</details>
</details>
<ul><li><i>environment</i>: <code>Centos + Jenkins 1.980.1</code></li></ul>
<details><summary><i>1 attachment</i></summary>

- [Jenkins-cannot-add-bug.png](https://issues.jenkins.io/secure/attachment/28092/Jenkins-cannot-add-bug.png)
  > ![Jenkins-cannot-add-bug.png](https://issues.jenkins.io/secure/attachment/28092/Jenkins-cannot-add-bug.png)
</details>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/danielbeck.png" title="danielbeck's avatar" /> danielbeck's <a href="https://issues.jenkins.io/browse/JENKINS-25836?focusedId=217080&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-217080">comment</a>:</i></summary>


<details><summary><sub><i>Raw content of original comment:</i></sub></summary>

<pre>
<p>I've seen this on my dev instance. It went away when I deleted the JENKINS_HOME, unfortunately I doubt this is an option for you.</p>

<p>Maybe delete extracted plugins and war to force them to be recreated.</p>

<p>Note that if you're asking for help, rather than reporting an issue, the jenkinsci-users list or #jenkins on Freenode would be better due to a larger audience. For tips how to ask, see <a href="http://www.catb.org/esr/faqs/smart-questions.html" class="external-link" target="_blank" rel="nofollow noopener">http://www.catb.org/esr/faqs/smart-questions.html</a> or <a href="https://wiki.jenkins-ci.org/display/JENKINS/How+to+report+an+issue" class="external-link" target="_blank" rel="nofollow noopener">https://wiki.jenkins-ci.org/display/JENKINS/How+to+report+an+issue</a></p>
</pre>
</details>

</details>
<p>I've seen this on my dev instance. It went away when I deleted the JENKINS_HOME, unfortunately I doubt this is an option for you.</p>

<p>Maybe delete extracted plugins and war to force them to be recreated.</p>

<p>Note that if you're asking for help, rather than reporting an issue, the jenkinsci-users list or #jenkins on Freenode would be better due to a larger audience. For tips how to ask, see <a href="http://www.catb.org/esr/faqs/smart-questions.html" class="external-link" target="_blank" rel="nofollow noopener">http://www.catb.org/esr/faqs/smart-questions.html</a> or <a href="https://wiki.jenkins-ci.org/display/JENKINS/How+to+report+an+issue" class="external-link" target="_blank" rel="nofollow noopener">https://wiki.jenkins-ci.org/display/JENKINS/How+to+report+an+issue</a></p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/danielbeck.png" title="danielbeck's avatar" /> danielbeck's <a href="https://issues.jenkins.io/browse/JENKINS-25836?focusedId=217116&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-217116">comment</a>:</i></summary>


<details><summary><sub><i>Raw content of original comment:</i></sub></summary>

<pre>
<blockquote><p>Note that if you're asking for help, rather than reporting an issue, the jenkinsci-users list or #jenkins on Freenode would be better due to a larger audience. For tips how to ask, see <a href="http://www.catb.org/esr/faqs/smart-questions.html" class="external-link" target="_blank" rel="nofollow noopener">http://www.catb.org/esr/faqs/smart-questions.html</a> or <a href="https://wiki.jenkins-ci.org/display/JENKINS/How+to+report+an+issue" class="external-link" target="_blank" rel="nofollow noopener">https://wiki.jenkins-ci.org/display/JENKINS/How+to+report+an+issue</a></p></blockquote>
</pre>
</details>

</details>
<blockquote><p>Note that if you're asking for help, rather than reporting an issue, the jenkinsci-users list or #jenkins on Freenode would be better due to a larger audience. For tips how to ask, see <a href="http://www.catb.org/esr/faqs/smart-questions.html" class="external-link" target="_blank" rel="nofollow noopener">http://www.catb.org/esr/faqs/smart-questions.html</a> or <a href="https://wiki.jenkins-ci.org/display/JENKINS/How+to+report+an+issue" class="external-link" target="_blank" rel="nofollow noopener">https://wiki.jenkins-ci.org/display/JENKINS/How+to+report+an+issue</a></p></blockquote>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/kevinhcross.jpg" title="kevinhcross's avatar" /> kevinhcross's <a href="https://issues.jenkins.io/browse/JENKINS-25836?focusedId=218949&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-218949">comment</a>:</i></summary>


<details><summary><sub><i>Raw content of original comment:</i></sub></summary>

<pre>
<p>We are using 1.580.2 and seeing the same issue.</p>

<p>I can confirm that using https instead of http also works on our instance.</p>

<p>We have another instance of Jenkins that we use to test upgrades (which is currently at the same version). The problem is not showing up on that instance (not yet at least). Th test instance has the same set of plugins. The only real difference is that it only has a few jobs enabled and runs them far less often.</p>
</pre>
</details>

</details>
<p>We are using 1.580.2 and seeing the same issue.</p>

<p>I can confirm that using https instead of http also works on our instance.</p>

<p>We have another instance of Jenkins that we use to test upgrades (which is currently at the same version). The problem is not showing up on that instance (not yet at least). Th test instance has the same set of plugins. The only real difference is that it only has a few jobs enabled and runs them far less often.</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/danielbeck.png" title="danielbeck's avatar" /> danielbeck's <a href="https://issues.jenkins.io/browse/JENKINS-25836?focusedId=223887&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-223887">comment</a>:</i></summary>


<details><summary><sub><i>Raw content of original comment:</i></sub></summary>

<pre>
<p>No response to comments in several months, so resolving as Cannot Reproduce.</p>

<p>Note that you absolutely must access Jenkins at the URL configured in global settings, otherwise it may fail to set up the session cookie, and the dropdowns require an active HTTP session.</p>
</pre>
</details>

</details>
<p>No response to comments in several months, so resolving as Cannot Reproduce.</p>

<p>Note that you absolutely must access Jenkins at the URL configured in global settings, otherwise it may fail to set up the session cookie, and the dropdowns require an active HTTP session.</p>
